doc/source/whatsnew/v0.23.1.txt

```diff
@@ -67,6 +67,11 @@ Categorical
 
 - Bug in :func:`pandas.util.testing.assert_index_equal` which raised ``AssertionError`` incorrectly, when comparing two :class:`CategoricalIndex` objects with param ``check_categorical=False`` (:issue:`19776`)
 
+Sparse
+^^^^^^
+
+- Bug in :attr:`SparseArray.shape` ignores ``fill_values`` (:issue:`21126`)
```

Comment:
can you expand on this slightly. maybe

Bzug in SparseArray.shape which previouslvy return a shape that only included the dense values.